crates/tako/src/internal/tests/test_scheduler_sn.rs

```diff
@@ -206,101 +206,101 @@ fn test_no_deps_distribute_with_balance() {
     core.sanity_check();
 }
 
-#[test]
-fn test_minimal_transfer_no_balance1() {
-    /*11  12
-      \  / \
-      13   14
-
-      11 - is big on W100
-      12 - is small on W101
-    */
-
-    let mut core = Core::default();
-    create_test_workers(&mut core, &[2, 2, 2]);
-    submit_example_1(&mut core);
-    start_and_finish_on_worker(&mut core, 11, 100, 10000);
-    start_and_finish_on_worker(&mut core, 12, 101, 1000);
-
-    let mut scheduler = create_test_scheduler();
-    let mut comm = create_test_comm();
-    scheduler.run_scheduling(&mut core, &mut comm);
-
-    comm.take_worker_msgs(100, 1);
-    comm.take_worker_msgs(101, 1);
-
-    check_task_has_worker(&core, 13, 100);
-    check_task_has_worker(&core, 14, 101);
-
-    comm.emptiness_check();
-    core.sanity_check();
-}
-
-#[test]
-fn test_minimal_transfer_no_balance2() {
-    /*11  12
-      \  / \
-      13   14
-
-      11 - is small on W100
-      12 - is big on W102
-    */
-    //setup_logging();
-    let mut core = Core::default();
-    create_test_workers(&mut core, &[2, 2, 2]);
-    submit_example_1(&mut core);
-    start_and_finish_on_worker(&mut core, 11, 100, 1000);
-    start_and_finish_on_worker(&mut core, 12, 101, 10000);
-
-    let mut scheduler = create_test_scheduler();
-    let mut comm = create_test_comm();
-    scheduler.run_scheduling(&mut core, &mut comm);
-
-    comm.take_worker_msgs(101, 2);
-
-    assert_eq!(
-        core.get_task(13.into()).get_assigned_worker().unwrap(),
-        WorkerId(101)
-    );
-    assert_eq!(
-        core.get_task(14.into()).get_assigned_worker().unwrap(),
-        WorkerId(101)
-    );
-
-    comm.emptiness_check();
-    core.sanity_check();
-}
-
-#[test]
-fn test_minimal_transfer_after_balance() {
-    /*11  12
-      \  / \
-      13   14
-
-      11 - is on W100
-      12 - is on W100
-    */
-
-    //setup_logging();
-    let mut core = Core::default();
-    create_test_workers(&mut core, &[1, 1]);
-    submit_example_1(&mut core);
-    start_and_finish_on_worker(&mut core, 11, 100, 10000);
-    start_and_finish_on_worker(&mut core, 12, 100, 10000);
-
-    let mut scheduler = create_test_scheduler();
-    let mut comm = create_test_comm();
-    scheduler.run_scheduling(&mut core, &mut comm);
-
-    comm.take_worker_msgs(100, 1);
-    comm.take_worker_msgs(101, 1);
-
-    check_task_has_worker(&core, 13, 100);
-    check_task_has_worker(&core, 14, 101);
-
-    comm.emptiness_check();
-    core.sanity_check();
-}
+// #[test]
```

Comment:
Come one, let's delete it :) We can find it in git history if needed.

Reply:
Ooookeeeyyy